### PR TITLE
HDDS-1489. Unnecessary log messages on console with Ozone shell.

### PR DESCRIPTION
**Pull request by swagle**
cc: @arp7

### CODE REVIEW

**Comment by arp7** [comment]
Instead of setting log threshold to WARN, would it be better to move the offending log messsages to DEBUG level?

**Comment by swagle** [comment]
@arp7 To change the log level of the message would need a Ratis Jira since it is, org.apache.ratis.grpc.client.GrpcClientProtocolClient.AsyncStreamObservers, logging this message.

File: hadoop-ozone/dist/src/main/compose/ozonefs/docker-config [hunk 1]
```diff
@@ -35,3 +35,4 @@ LOG4J.PROPERTIES_log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 LOG4J.PROPERTIES_log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
 LOG4J.PROPERTIES_log4j.logger.org.apache.hadoop.util.NativeCodeLoader=ERROR
 LOG4J.PROPERTIES_log4j.logger.org.apache.ratis.conf.ConfUtils=WARN
+LOG4J.PROPERTIES_log4j.logger.org.apache.ratis.grpc.client.GrpcClientProtocolClient=WARN
```
Comment (shwetayakkali) [inline hadoop-ozone/dist/src/main/compose/ozonefs/docker-config]: checkstyle warning to add a new line at the end of file.

File: hadoop-ozone/dist/src/main/compose/ozoneperf/docker-config [hunk 2]
```diff
@@ -36,3 +36,4 @@ LOG4J.PROPERTIES_log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH
 LOG4J.PROPERTIES_log4j.logger.org.apache.hadoop.util.NativeCodeLoader=ERROR
 LOG4J.PROPERTIES_log4j.logger.org.apache.ratis.conf.ConfUtils=WARN
 LOG4J.PROPERTIES_log4j.logger.org.apache.hadoop.security.ShellBasedUnixGroupsMapping=ERROR
+LOG4J.PROPERTIES_log4j.logger.org.apache.ratis.grpc.client.GrpcClientProtocolClient=WARN
```
Comment (shwetayakkali) [inline hadoop-ozone/dist/src/main/compose/ozoneperf/docker-config]: checkstyle warning here to add a new line at the end of file.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 33 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| 0 | shelldocs | 0 | Shelldocs was not available. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ trunk Compile Tests _ |
| 0 | mvndep | 48 | Maven dependency ordering for branch |
| +1 | mvninstall | 433 | trunk passed |
| +1 | compile | 211 | trunk passed |
| +1 | mvnsite | 0 | trunk passed |
| +1 | shadedclient | 740 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 131 | trunk passed |
||| _ Patch Compile Tests _ |
| 0 | mvndep | 16 | Maven dependency ordering for patch |
| +1 | mvninstall | 426 | the patch passed |
| +1 | compile | 208 | the patch passed |
| +1 | javac | 208 | the patch passed |
| +1 | mvnsite | 0 | the patch passed |
| +1 | shellcheck | 0 | There were no new shellcheck issues. |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 640 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 124 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 141 | hadoop-hdds in the patch failed. |
| -1 | unit | 952 | hadoop-ozone in the patch failed. |
| +1 | asflicense | 47 | The patch does not generate ASF License warnings. |
| | | 4322 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.ozone.client.rpc.TestOzoneAtRestEncryption |
|   | hadoop.ozone.client.rpc.TestOzoneRpcClient |
|   | hadoop.hdds.scm.pipeline.TestRatisPipelineProvider |
|   | hadoop.ozone.container.common.statemachine.commandhandler.TestBlockDeletion |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-797/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/797 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient shellcheck shelldocs |
| uname | Linux 949ab61dfba0 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 93f2283 |
| Default Java | 1.8.0_191 |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-797/1/artifact/out/patch-unit-hadoop-hdds.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-797/1/artifact/out/patch-unit-hadoop-ozone.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-797/1/testReport/ |
| Max. process+thread count | 4370 (vs. ulimit of 5500) |
| modules | C: hadoop-ozone/dist hadoop-ozone/ozonefs U: hadoop-ozone |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-797/1/console |
| versions | git=2.7.4 maven=3.3.9 shellcheck=0.4.6 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 30 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| 0 | shelldocs | 1 | Shelldocs was not available. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ trunk Compile Tests _ |
| 0 | mvndep | 37 | Maven dependency ordering for branch |
| +1 | mvninstall | 390 | trunk passed |
| +1 | compile | 210 | trunk passed |
| +1 | mvnsite | 0 | trunk passed |
| +1 | shadedclient | 774 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 128 | trunk passed |
||| _ Patch Compile Tests _ |
| 0 | mvndep | 25 | Maven dependency ordering for patch |
| +1 | mvninstall | 428 | the patch passed |
| +1 | compile | 210 | the patch passed |
| +1 | javac | 210 | the patch passed |
| +1 | mvnsite | 0 | the patch passed |
| +1 | shellcheck | 0 | There were no new shellcheck issues. |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 656 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 134 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 139 | hadoop-hdds in the patch failed. |
| -1 | unit | 845 | hadoop-ozone in the patch failed. |
| -1 | asflicense | 43 | The patch generated 1 ASF License warnings. |
| | | 4873 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-797/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/797 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient shellcheck shelldocs |
| uname | Linux 27c7ba8ec768 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 8ecbf61 |
| Default Java | 1.8.0_191 |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-797/2/artifact/out/patch-unit-hadoop-hdds.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-797/2/artifact/out/patch-unit-hadoop-ozone.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-797/2/testReport/ |
| asflicense | https://builds.apache.org/job/hadoop-multibranch/job/PR-797/2/artifact/out/patch-asflicense-problems.txt |
| Max. process+thread count | 4282 (vs. ulimit of 5500) |
| modules | C: hadoop-ozone/dist hadoop-ozone/ozonefs U: hadoop-ozone |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-797/2/console |
| versions | git=2.7.4 maven=3.3.9 shellcheck=0.4.6 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 24 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| 0 | shelldocs | 0 | Shelldocs was not available. |
| +1 | @author | 1 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ trunk Compile Tests _ |
| 0 | mvndep | 54 | Maven dependency ordering for branch |
| +1 | mvninstall | 412 | trunk passed |
| +1 | compile | 231 | trunk passed |
| +1 | mvnsite | 0 | trunk passed |
| +1 | shadedclient | 814 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 128 | trunk passed |
||| _ Patch Compile Tests _ |
| 0 | mvndep | 16 | Maven dependency ordering for patch |
| +1 | mvninstall | 403 | the patch passed |
| +1 | compile | 221 | the patch passed |
| +1 | javac | 221 | the patch passed |
| +1 | mvnsite | 0 | the patch passed |
| +1 | shellcheck | 0 | There were no new shellcheck issues. |
| +1 | whitespace | 1 | The patch has no whitespace issues. |
| +1 | shadedclient | 706 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 127 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 151 | hadoop-hdds in the patch failed. |
| -1 | unit | 1126 | hadoop-ozone in the patch failed. |
| -1 | asflicense | 38 | The patch generated 1 ASF License warnings. |
| | | 5185 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-797/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/797 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient shellcheck shelldocs |
| uname | Linux 59a3b19cf00a 4.4.0-139-generic #165~14.04.1-Ubuntu SMP Wed Oct 31 10:55:11 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / eb9c890 |
| Default Java | 1.8.0_191 |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-797/3/artifact/out/patch-unit-hadoop-hdds.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-797/3/artifact/out/patch-unit-hadoop-ozone.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-797/3/testReport/ |
| asflicense | https://builds.apache.org/job/hadoop-multibranch/job/PR-797/3/artifact/out/patch-asflicense-problems.txt |
| Max. process+thread count | 4123 (vs. ulimit of 5500) |
| modules | C: hadoop-ozone/dist hadoop-ozone/ozonefs U: hadoop-ozone |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-797/3/console |
| versions | git=2.7.4 maven=3.3.9 shellcheck=0.4.6 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Pull request merged**